esx_service/utils/vmdk_utils.py

```diff
@@ -73,10 +73,11 @@ def init_datastoreCache(force=False):
     from local ESX host. force=True will force it to ignore current cache
     and force init
     """
+    global datastores
+
```

Comment:
The comment on @lipingxue's change was to place this line under the "with....." statement below :-)